bridges/StockFilingsBridge.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+class StockFilingsBridge extends FeedExpander {
+	const MAINTAINER = 'captn3m0';
+	const NAME = 'SEC Stock filings';
+	const URI = 'https://www.sec.gov/edgar/searchedgar/companysearch.html';
+	const CACHE_TIMEOUT = 3600; // 1h
+	const DESCRIPTION = 'Tracks SEC Filings for a single company';
+	const SEARCH_URL = 'https://www.sec.gov/cgi-bin/browse-edgar?owner=exclude&action=getcompany&CIK=';
+	const WEBSITE_ROOT = 'https://www.sec.gov';
+
+	const PARAMETERS = array(
+		array(
+		'ticker' => array(
+			'name' 			=> 'cik',
```

Comment:
I know this is nitpicking, but your alignment tabs don't align when using different tab width (i.e. 2 instead of 4). Generally, alignment tabs should only be used at the beginning of a line.